### ISSUE TITLE
Question about experiment setting

### ISSUE THREAD

**Issue by MooreManor** [ShirleyMaxx/VirtualMarker#4]
Hello! @ShirleyMaxx 

VirtualMarker is nice work. I have a question about the experiment setting about 3dpw in table 2. Most of the prior works use the mixed datasets to train and then test on 3dpw test set, while virtualmarker is first trained on mixed datasets and then finetuned on 3dpw train set. Do you have the results for virtualmarker trained on mixed datasets which include 3dpw train set instead of fine-tuning for a second stage?

**Comment by ShirleyMaxx** [comment]
Hi, thanks for your interest. As you said, we report the results by finetuning on the 3DPW dataset for faster training, as [MeshTransformer](https://github.com/microsoft/MeshTransformer/blob/main/docs/EXP.md#training-with-3dpw-dataset) does. We haven't tried to train on mixed datasets which include 3DPW train set. Empirically, if training on mixed datasets, increasing the proportion of 3DPW train set by resampling may be needed for higher 3DPW performance.
